<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_driver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>    운전자</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_drivers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>박중후</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>김태산</string>
    </property>
   </item>
  <item><property name="text"><string>정의석</string></property></item><item><property name="text"><string>류정환</string></property></item><item><property name="text"><string>오기성</string></property></item></widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineedit_add">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_new">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>   신규 운전자 입력</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
 </widget>
 <resources />
 <connections />
</ui>